phoneApp/user.js

```diff
@@ -0,0 +1,96 @@
+class User {
+	constructor(user) {
+		this.user = user;
+		this.options = [
+			{option: 'message', class: 'glyphicon-comment'},
```

Comment:
the same about options, we don't need such data inside property, better to make it just static